### ISSUE TITLE
Pressing Cmd+A 2 times in widget areas selects all of the widget areas rather than the widgets

### ISSUE THREAD

**Issue by kevin940726** [WordPress/gutenberg#28267]
## Description

Pressing <kbd>Cmd</kbd> + <kbd>a</kbd> 2 times in widget areas selects all the widget areas, but not all the widgets inside the currently selected widget area. See the below screenshot for more info.

## Step-by-step reproduction instructions

1. Go to the widgets screen
2. Select a widget area
3. Focus on a random widget inside the widget area
4. Press <kbd>Cmd</kbd> + <kbd>a</kbd> 2 times
5. Notice all the widget areas get selected

## Expected behaviour

It should instead selects all the widgets inside the currently focused widget area.

## Actual behaviour

All the widget areas and all the widgets get selected

## Screenshots or screen recording (optional)

https://user-images.githubusercontent.com/7753001/104884538-7d327e80-59a1-11eb-9932-ac8283953a77.mp4

## WordPress information
- WordPress version: 5.7-alpha-49956 (`wp-env`)
- Gutenberg version: master branch
- Are all plugins except Gutenberg deactivated? Yes
- Are you using a default theme (e.g. Twenty Twenty-One)? Yes

## Device information
- Device:  Desktop
- Operating system:  macOS
- Browser:  Chrome 87

**Comment by draganescu** [comment]
*Notes*

The shortcut Primary + A twice will trigger `multiSelect` between the first and the last block in `getBlockOrder()` a selector in the block editor store which accepts an optional `rootClientId` "of the block list for which the order should be returned, defaulting to the top-level block order".

It feels like the `getBlockOrder` selector should either default to parent as  `rootClientId` or we should rebind the select all shortcut in the widgets editor.

**Comment by talldan** [comment]
Looks like this would also be fixed/improved by https://github.com/WordPress/gutenberg/pull/31859, though it would still be possible to select widget areas.

**Comment by noisysocks** [comment]
Assigning @talldan because IIRC he's taking over #30462.
